docs/application/native/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1186 @@
+# Voice Control Manager
+
+
+Voice control manager features allow you to record voice and give responses for the recognized voice commands. You can register general and system voice commands such as "power on", "power off", "music play", "music stop", and so on. In addition, you can start and stop voice recording. When the voice recording is finished, you can receive multiple recognition results such as Automatic Speech Recognition (ASR) and matched commands from the commands list, which is registered by the application using the voice control client.
```

Comment:
**Original :** You can register general and system voice commands such as "power on", "power off", "music play", "music stop", and so on.

1. **Suggested :** You can register general, and system voice commands 

   In this sentence, it means you register "general voice commands" and "system voice commands".
   So adding a comma before "and" doesn't seem good to me.

2. Also, "power on", "power off", "music play" and "music stop" are commands, not UI elements or else.
   So I think "" is good to apply to the "commands".

3. Changing "In addition" to "Also".
   @annie-samsung This kind of updating suggestion is less effective comparing to the time used to update the file & to check the comments, so how about reducing this kind of review? It takes too much time to cross check the comments but doesn't improve the quality of sentence a lot comparing to the effort.


**Reference page :** https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/guides/text-input/voice-control-manager.md

Reply:
> **Original :** You can register general and system voice commands such as "power on", "power off", "music play", "music stop", and so on.
> 
> 1. **Suggested :** You can register general, and system voice commands
>    In this sentence, it means you register "general voice commands" and "system voice commands".
>    So adding a comma before "and" doesn't seem good to me.
> 2. Also, "power on", "power off", "music play" and "music stop" are commands, not UI elements or else.
>    So I think "" is good to apply to the "commands".
> 3. Changing "In addition" to "Also".
>    @annie-samsung This kind of updating suggestion is less effective comparing to the time used to update the file & to check the comments, so how about reducing this kind of review? It takes too much time to cross check the comments but doesn't improve the quality of sentence a lot comparing to the effort.
> 
> **Reference page :** https://github.com/Samsung/tizen-docs/blob/master/docs/application/dotnet/guides/text-input/voice-control-manager.md

@mijin85cho , I agree, @nawab-samsung please provide comments only if you see any grammatical issues,, any typos, missing information, or if there are any style issues. With the just two of us reviewing now, this would help us save time and also help to increase the turn around time and close the PR faster.